[MWAN] Combines tunnels info (#8714)
cloudflare/cloudflare-docs: content/magic-transit/_partials/_tunnels-encapsulation-opening.md

- **Auth Exchange** (`IKE_AUTH`): Using the secure tunnel established in the initial exchange, IKE peers mutually authenticate each other. After authentication, the IKE security association (SA) is established. Next, the peers negotiate and establish an IPsec tunnel, known as a Child SA.

{{<Aside type="note">}}It is important to note that the IKE SA and the Child SA are separate things, each with their own parameters. The Child SA is the actual IPsec tunnel where user traffic flows (that is, the dataplane). The IKE SA is used to set up and manage the Child SA.{{</Aside>}}

In summary, an IKE SA is created that uses certain cryptographic transforms. That IKE SA is then used to create a Child SA which itself uses certain cryptographic transforms. The configuration section below details which of these transforms for IKE SAs and Child SAs are currently supported by $1.

### Supported configuration parameters

#### IKE SA

> This is sometimes referred to as “Phase 1” due to language from IKEv1.

Reviewer: ```suggestion
This is sometimes referred to as Phase 1 as per IKEv1 language.
```

Don't think you need "" as per style guide and below on line 107 you also say `... referred to as Authentication`.

- **Encryption**
  - AES-GCM-16 with 128-bit or 256-bit key length
  - AES-CBC with 256-bit key length

- **Integrity** (sometimes referred to as Authentication)
  - SHA2-256

- **Diffie-Hellman group**:
  - DH group 14 (2048-bit MODP group)
  - DH group 5 (1536-bit MODP group)

- **Pseudorandom function (PRF)** (not to be confused with PFS. PRF is often not a configurable setting.)
  - SHA2-256
  - SHA2-384
  - SHA2-512

#### IPsec

> The Child SA. Sometimes referred to as “Phase 2” due to language from IKEv1.

Reviewer: ```suggestion
The Child SA. Sometimes referred to as Phase 2 as per IKEv1 language.
```

- **Encryption**: 
  - AES-GCM-16 with 128-bit or 256-bit key length
  - AES-CBC with 128-bit or 256-bit key length

- **Integrity** (sometimes referred to as Authentication.)
  - SHA2-256
  - SHA-1
  {{<Aside type="note">}}When using AES-GCM-16, an integrity algorithm is not required because AES GCM includes integrity checking (since it is an AEAD algorithm). Even when using an AEAD algorithm, however, some routers still require an integrity algorithm to be selected.{{</Aside>}}

> - **PFS group** (sometimes referred to as "Phase 2 Diffie-Hellman Group". Not to be confused with PRF.)

Reviewer: ```suggestion
- **PFS group** (sometimes referred to as Phase 2 Diffie-Hellman Group. Not to be confused with PRF.)
```

  - DH group 14 (2048-bit MODP group)
  - DH group 5 (1536-bit MODP group)

#### Required configuration parameters

- The IKE version must be IKEv2.
- The IKE authentication method must be Pre-Shared Key (PSK).
- [Anti-replay protection]($3) must be disabled.

> - If your router is behind NAT and requires NAT traversal (NAT-T), then your router must initiate IKE communication on port 4500. Most devices support configuring NAT-T to begin on port `4500` (exceptions include at least some versions of the Cisco ASA). NAT-T is not supported for IKE sessions which begin on port `500` and then switch to port `4500`.

Reviewer: ```suggestion
- If your router is behind NAT and requires NAT traversal (NAT-T), then your router must initiate IKE communication on port `4500`. Most devices support configuring NAT-T to begin on port `4500` (exceptions include at least some versions of the Cisco ASA). NAT-T is not supported for IKE sessions which begin on port `500` and then switch to port `4500`.
```
